packages/generators/remove-generator.js

```diff
@@ -1,3 +1,119 @@
 const Generator = require("yeoman-generator");
+const path = require("path");
+const fs = require("fs");
+const { List } = require("@webpack-cli/webpack-scaffold");
 
-module.exports = class RemoveGenerator extends Generator {};
+const PROP_TYPES = require("@webpack-cli/utils/prop-types");
+
+/**
+ *
+ * Generator for removing properties
+ * @class	RemoveGenerator
+ * @extends	Generator
+ * @returns	{Void} After execution, transforms are triggered
+ *
+ */
+
+module.exports = class RemoveGenerator extends Generator {
+	constructor(args, opts) {
+		super(args, opts);
+		this.configuration = {
+			config: {
+				webpackOptions: {},
+			}
+		};
+
+		let configPath = path.resolve(process.cwd(), "webpack.config.js");
+		const webpackConfigExists = fs.existsSync(configPath);
+		if (!webpackConfigExists) {
+			configPath = null;
+			// end the generator stating webpack config not found or to specify the config
+		}
+		this.webpackOptions = require(configPath);
+		this.configuration.config.webpackOptions = this.webpackOptions;
+	}
+
+	getPropTypes() {
+		return Object.keys(this.webpackOptions);
+	}
+
+	getModuleLoaders() {
+		if (this.webpackOptions.module && this.webpackOptions.module.rules) {
+			return this.webpackOptions.module.rules.map(rule => rule ? rule.loader : null);
```

Comment:
Add a filter here cause some props might be null and therefore we shouldn't add more than needed to the yeoman file. Don't you think?